sdk/keyvault/samples/sharelink/Generated/AzureSecurityKeyVaultStorageModelFactory.cs

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This is a sample and doesn't need this. I'd rather not have to maintain it or complicate the sample with unnecessary code.

Reply:
If this can't be elided, can you at least use the `CodeModelAttribute` to make it internal. I'd rather not have it in the sample, but not exposing it is next best.